Adding vial for a_dux and cuttlefish.

jmighion opened the pull request:
Adding vial for a_dux, clog, and cuttlefish.
@smores56 @tapioki made the boards.

Kept as much of the qmk as possible for [a dux](https://github.com/qmk/qmk_firmware/tree/master/keyboards/a_dux) and the [clog](https://github.com/qmk/qmk_firmware/pull/23009)

Formatted with [qmkfmt](https://github.com/rcorre/qmkfmt)

jmighion committed:
Adding vial for a_dux, clog, and cuttlefish.

jmighion committed:
Removing VIAL_INSECURE

xyzz commented:
> Kept as much of the qmk as possible for a dux and the clog

can you elaborate what you mean by this? if the keyboard is being submitted to qmk (as I see from the linked PR) then it should have exactly the same contents as in qmk, outside of the vial keymap, and in that case we have to wait for qmk to merge it first. otherwise, any difference between qmk and vial-qmk would cause conflicts in the next breaking changes merge which are annoying to resolve

jmighion committed:
Fixing QK_GESC for default qmk.

jmighion committed:
Mirroring clog's default keymap formatting.

jmighion commented:
Fixed the build issue. The only difference this default qmk setup for the clog was the formatting of the keymap.c, which I've reverted to match exactly what's in that PR. Should I just separate out the clog from this PR to get the others in without waiting for qmk to merge that other PR?

xyzz commented:
> Should I just separate out the clog from this PR to get the others in without waiting for qmk to merge that other PR?

sure, that's fine

jmighion committed:
Removing clog until it's in upstream qmk

jmighion commented:
@xyzz Done! I'll have another PR up whenever clog get merged into qmk

xyzz commented on keyboards/a_dux/keymaps/default/keymap.c:
```diff
@@ -1,40 +1,34 @@
-// Copyright 2022 @filterpaper
```
this change should be reverted to match upstream qmk

xyzz commented on keyboards/cuttlefish/readme.md:
```diff
@@ -0,0 +1,26 @@
+# The Clog
```
the readme isn't for the right keyboard, you can fix or just delete it altogether

jmighion replied:
Made on in the style of the other

jmighion committed:
Fixing diff between qmk, broken vial keymap, and readme.

jmighion committed:
Missing newline EOF
